packages/client/src/components/message-group.js

```diff
@@ -45,8 +45,11 @@ const component = (state, emit, group) => {
       </div>
 
       ${group.messages.map(msg => {
-        const el = html`<div class='Message' id=${'msg-' + msg.id}>
-          ${raw(mrk(state)(msg.text).html())}
+        const mrked = mrk(state)(msg.text)
+        const bigEmotes = mrked.tokens.length <= 16 && !mrked.tokens.find(t => t.name !== 'emote')
```

Comment:
This is a pretty cool way of using `mrk`.

Reply:
Yep! Part of the reason for its existence is access to the token stream.